pennylane/liealg/horizontal_cartan_subalgebra.py

```diff
@@ -403,8 +404,8 @@ def op_to_adjvec(
     .. seealso:: :func:`~adjvec_to_op`
 
     Args:
-        ops (Iterable[Union[PauliSentence, Operator, TensorLike]]): List of operators to decompose
-        basis (Iterable[Union[PauliSentence, Operator, TensorLike]]): Operator basis
+        ops (Iterable[Union[PauliSentence, Operator, TensorLike]]): List of operators to decompose.
+        basis (Iterable[Union[PauliSentence, Operator, TensorLike]]): Operator basis.
```

Comment:
We go back and forth on [this](https://docs.pennylane.ai/en/stable/development/guide/documentation.html#punctuation-and-spelling) in PR reviews.
```suggestion
        ops (Iterable[Union[PauliSentence, Operator, TensorLike]]): list of operators to decompose
        basis (Iterable[Union[PauliSentence, Operator, TensorLike]]): operator basis
```